tests/test_ex1.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+import pytest
+from pytest_mock import MockerFixture
+import src.ex1 as ex1
+
+
+
+
+def test_ex1()->None:
+
+    days = [
```

Comment:
potresti prendere questa variabile dall'altro file, però nell'altro file questa variabile è dentro la funzione, dovresti scriverla fuori la funzione in modo da poterla condividere con altri file